wiki/reinstalling-microsoft-store.md

```diff
@@ -0,0 +1,65 @@
+# Reinstalling the Microsoft Store
+
+The Microsoft Store is used to download applications. But it may sometimes glitch out and disappear. This article will cover the steps of reinstalling it.
+
+## Method 1: Resetting the Microsoft Store cache
+
+There are two methods to clear out the Microsoft Store cache. 
+
+### Method 1: Resetting the Microsoft Store through the Settings App
+
+1. Open the Settings app. You can do this by searching for it in the Start menu, pressing <kbd>Win</kbd> + <kbd>I</kbd>, or pressing the Settings icon above the power button.
+
+    ![](img/reinstalling-microsoft-store/opensettings.png)
+
+2. Select "Apps".
+
+    <img src="img/reinstalling-microsoft-store/apps.png" style="width:300px;">
+
+3. Find "Microsoft Store" and click advanced options.
+
+    <img src="img/reinstalling-microsoft-store/microsoftstore.png" style="width:500px">
+
+> If you can't find the Microsoft Store here, you will have to reinstall it completely. Follow Method 2 to reinstall the Microsoft Store
+
+4. Scroll down and click the reset button. Then wait for a check mark to appear.
+
+    <img src="img/reinstalling-microsoft-store/resetmicrosoftstore.png" style="width:500px">
+
+### Method 2: Resetting the Microsoft Store through WSReset
+
+> The WSReset function is outdated and may not function correctly. If Method 1 of resetting the Microsoft Store cache doesn't work, try reinstalling the Microsoft Store. This method is still available.
```

Comment:
"...and may not function correctly in the latest versions of Windows 10"

"If Method 1 for"